app/src/main/java/nerd/tuxmobil/fahrplan/congress/settings/SettingsFragment.kt

```diff
@@ -92,4 +147,10 @@ class SettingsFragment : PreferenceFragment() {
         val intent = Intent(Settings.ACTION_APP_NOTIFICATION_SETTINGS).apply { putExtras(extras) }
         startActivity(intent)
     }
+
+    /**
+     * See [findPreference].
+     */
+    private fun <T : Preference> requirePreference(key: CharSequence): T = findPreference(key)!!
```

Comment:
Having a separate function for this opens the opportunity to provide a helpful error message when the preference couldn't be found.

Reply:
:heavy_check_mark: Changed in 23b10ca4822a133ed18915842987e132b4a9205a.